test/fixtures/_generateCoverage.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+// Generates the test/fixtures/coverage.js file, not otherwise used in the
+// tests.
+
+var fs = require('fs')
+var path = require('path')
+
+var _ = require('lodash')
+var rimraf = require('rimraf')
+
+var NYC = require('../../')
+
+// Load the 'branching' source map fixture.
+var fixture = require('source-map-fixtures').inline('branching')
+
+// Prevent pollution from earlier nyc runs.
+var tempDirectory = path.join(__dirname, '.nyc_output')
+rimraf.sync(tempDirectory)
+
+// Inject nyc into this process.
+var nyc = (new NYC({
+  cwd: path.join(__dirname, '..', '..'),
+  tempDirectory: tempDirectory
+})).wrap()
+// Override the exclude option, source-map-fixtures is inside node_modules but
+// should not be excluded when generating the coverage report.
+nyc.exclude = []
+
+// Require the fixture so nyc can instrument it, then run it so there's code
+// coverage.
+fixture.require().run()
+
+// Write the coverage file so reports can be loaded.
+nyc.writeCoverageFile()
+
+var reports = _.values(nyc._loadReports()[0])
```

Comment:
this is a great approach for generating the fixtures, and I love the module `source-map-fixtures`.